ui/platform_window/x11/x11_window_base.cc

```diff
@@ -260,8 +261,8 @@ void X11WindowBase::SetCapture() {}
 void X11WindowBase::ReleaseCapture() {}
 
 void X11WindowBase::ToggleFullscreen() {
-  SetWMSpecState(!is_fullscreen_,
-                 gfx::GetAtom("_NET_WM_STATE_FULLSCREEN"), None);
+  SetWMSpecState(!is_fullscreen_, gfx::GetAtom("_NET_WM_STATE_FULLSCREEN"),
```

Comment:
if it is a coding style tweak but unrelated to this patch, I would leave it out.